src/bin/ion/commands/inspect.rs

```diff
@@ -393,29 +398,64 @@ impl<'a, 'b> IonInspector<'a, 'b> {
     /// and then reset it upon completion.
     fn with_style(
         &mut self,
-        style: ColorSpec,
-        write_fn: impl FnOnce(&mut CommandOutput) -> Result<()>,
+        mut style: ColorSpec,
+        write_fn: impl FnOnce(&mut Self) -> Result<()>,
```

Comment:
🪧 Passing `Self` means that the closure has access to `IonInspector`'s fields when formatting comments.